test/OpenFeature.Benchmarks/OpenFeatureClientBenchmarks.cs

```diff
@@ -0,0 +1,175 @@
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using AutoFixture;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+using OpenFeature.Model;
+
+namespace OpenFeature.Benchmark
+{
+    [MemoryDiagnoser]
+    [SimpleJob(RuntimeMoniker.Net60, baseline: true)]
+    [RPlotExporter]
+    public class OpenFeatureClientBenchmarks
+    {
+        private readonly string _clientName;
+        private readonly string _clientVersion;
+        private readonly string _flagName;
+        private readonly bool _defaultBoolValue;
+        private readonly string _defaultStringValue;
+        private readonly int _defaultIntegerValue;
+        private readonly double _defaultDoubleValue;
+        private readonly Value _defaultStructureValue;
+        private readonly FlagEvaluationOptions _emptyFlagOptions;
+
+        public OpenFeatureClientBenchmarks()
+        {
+            var fixture = new Fixture();
+            _clientName = fixture.Create<string>();
+            _clientVersion = fixture.Create<string>();
+            _flagName = fixture.Create<string>();
+            _defaultBoolValue = fixture.Create<bool>();
+            _defaultStringValue = fixture.Create<string>();
+            _defaultIntegerValue = fixture.Create<int>();
+            _defaultDoubleValue = fixture.Create<double>();
+            _defaultStructureValue = fixture.Create<Value>();
+            _emptyFlagOptions = new FlagEvaluationOptions(ImmutableList<Hook>.Empty, ImmutableDictionary<string, object>.Empty);
+        }
+
+        [Benchmark]
+        public async Task OpenFeatureClient_GetBooleanValue_WithoutEvaluationContext_WithoutFlagEvaluationOptions()
+        {
+            Api.Instance.SetProvider(new NoOpFeatureProvider());
```

Comment:
Should setting the provider and getting the client be setup outside the bench?

Reply:
Updated. Thanks for comment